compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -324,7 +324,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     ungated!(export_name, Normal, template!(NameValueStr: "name"), FutureWarnPreceding),
     ungated!(link_section, Normal, template!(NameValueStr: "name"), FutureWarnPreceding),
     ungated!(no_mangle, Normal, template!(Word), WarnFollowing),
-    ungated!(used, Normal, template!(Word), WarnFollowing),
+    ungated!(used, Normal, template!(Word, List: "used"), WarnFollowing),
```

Comment:
I'd use
```suggestion
    ungated!(used, Normal, template!(Word, List: "compiler|linker"), WarnFollowing),
```
here, similar to what `inline` does. Not sure where exactly this string is being used though.

Reply:
Thanks!